lms/static/scripts/frontend_apps/components/LaunchErrorDialog.tsx

```diff
@@ -611,17 +615,51 @@ export default function LaunchErrorDialog({
       );
 
     case 'error-reporting-submission':
-      // nb. There is no retry action here as we just suggest reloading the entire
-      // page.
       return (
         <ErrorModal
           {...defaultProps}
           onRetry={undefined}
-          title="Something went wrong"
+          onCancel={onCancel}
```

Comment:
Wording from:

https://hypothes-is.slack.com/archives/C2C2U40LW/p1716385676586839?thread_ts=1716217003.396489&cid=C2C2U40LW


https://hypothes-is.slack.com/archives/C2C2U40LW/p1716385676586839?thread_ts=1716217003.396489&cid=C2C2U40LW


Although I'd vote to merge any version  with the code changes to enable the button & background and then do a another PR if the text needs to be revisited.